From 6ae7d71893dff9aa4b4cd92ec5547be62c318b09 Mon Sep 17 00:00:00 2001
From: Dylan Baker <dylan.c.baker@intel.com>
Date: Fri, 16 Apr 2021 09:24:27 -0700
Subject: [PATCH] meson: OpenMP is supposed to be optional

The code around it checks that it is found, so clearly it was meant to
be optional, not arequired.

Fixes: cd2832ee5107201493d59f70983f162653e53c7d
       ("meson: add an optional OpenMP dependency for AMD tests")
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 2849ae00492..7c1cc07fd59 100644
--- a/meson.build
+++ b/meson.build
@@ -1872,7 +1872,7 @@ dep_openmp = null_dep
 
 # Even if we find OpenMP, Gitlab CI fails to link with gcc/i386 and clang/anyarch.
 if host_machine.cpu_family() == 'x86_64' and cc.get_id() == 'gcc'
-  dep_openmp = dependency('openmp')
+  dep_openmp = dependency('openmp', required : false)
   if dep_openmp.found()
     pre_args += ['-DHAVE_OPENMP']
   endif
-- 
GitLab

